build.gradle

```diff
@@ -30,4 +31,8 @@ compileTestKotlin {
 
 application {
     mainClassName = 'MainKt'
+}
+
+ktfmt {
+    googleStyle()
```

Comment:
Is there anything else we need to configure for this?

Reply:
It is interesting this works for you. We have to use `spotless` plugin to make `ktfmt` work with Gradle. 
https://github.com/stellar/java-stellar-anchor-sdk/blob/main/build.gradle.kts#L61 

Does this reformat your code when you run `./gradlew build`?

Reply:
@lijamie98 Yes, it seems to work. This is what I get when I run `./gradlew build` with the wrong formatting. Is that expected?

![image](https://user-images.githubusercontent.com/7346473/184680746-fbca04ce-7a95-4d0c-9c84-c0e8dc5a84d0.png)

Reply:
IntelliJ has [ktfmt plugin](https://plugins.jetbrains.com/plugin/14912-ktfmt) that may help to work with the IDE better.

Reply:
Looks like your Gradle setting only identifies the problem rather than fixing it. You will need to apply that.

In [AP (Anchor Platform)](https://github.com/stellar/java-stellar-anchor-sdk/blob/main/build.gradle.kts#L39), we use `spotless` plugin to apply the formatting. It can be a good reference for you.

Reply:
:tada: It's fixing formatting with `spotless` now. Thank you!